app/controllers/challenge_requests_controller.rb

```diff
@@ -21,17 +21,17 @@ def index
     set_sort_order
     
     # actual content, do the efficient method unless we need the full query
-    
+
     if @sort_column == "fandom"
       query = "SELECT prompts.*, GROUP_CONCAT(tags.name) AS tagnames FROM prompts INNER JOIN set_taggings ON prompts.tag_set_id = set_taggings.tag_set_id 
       INNER JOIN tags ON tags.id = set_taggings.tag_id 
       WHERE prompts.type = 'Request' AND tags.type = 'Fandom' AND prompts.collection_id = " + @collection.id.to_s + " GROUP BY prompts.id ORDER BY tagnames " + @sort_direction
       @requests = Prompt.paginate_by_sql(query, page: params[:page], per_page: ArchiveConfig.ITEMS_PER_PAGE)
     elsif @sort_column == "prompter" && !@collection.prompts.where(anonymous: true).exists?
       @requests = @collection.prompts.where("type = 'Request'").
-                              joins(challenge_signup: :pseud).
-                              order("pseuds.name #{@sort_direction}").
-                              paginate(page: params[:page])
+        joins(challenge_signup: :pseud).
+        order(["pseuds.name ?", @sort_direction]).
```

Comment:
The automated tests don't catch it because this line isn't covered, but I believe this doesn't work. I believe it encloses the sort direction in quotes, which isn't allowed.